src/main/java/com/minecolonies/coremod/tileentities/TileEntityColonyBuilding.java

```diff
@@ -23,6 +23,7 @@
      * NBTTag to store the colony id.
      */
     private static final String TAG_COLONY = "colony";
+    private static final String TAG_MIRROR = "mirror";
```

Comment:
I don't think we need to store this in the TileEntity, why do mirroring any different than we do rotation?